src/main/java/io/cdap/plugin/gcp/bigquery/source/BigQuerySourceConfig.java

```diff
@@ -171,6 +180,26 @@ public String getPartitionTo() {
     return Strings.isNullOrEmpty(partitionTo) ? null : partitionTo;
   }
 
+  @Nullable
+  public String getFilter() {
+    if (filter != null) {
+      filter = filter.trim();
+      if (filter.isEmpty()) {
+        return null;
+      }
+      // remove the WHERE from the filter
```

Comment:
Why do we need user to specify WHERE can the user only specify the expression of WHERE. This makes it more cleaner.

Reply:
@nitinmotgi the user does not need to specify the WHERE keyword in the BigQuery Source plugin, he/she needs only to specify the expression of WHERE. We have put this validation just in case someone copy/paste or accidentally puts the WHERE keyword at the beginning.

Reply:
can you update the comment